conduct/artifacts/CODE_OF_CONDUCT_PATCH.patch

```diff
@@ -0,0 +1,41 @@
+--- ./coc-expected.md	2023-03-07 12:46:25
++++ ./coc-current.md	2023-03-10 09:20:48
+@@ -43,16 +43,15 @@
+ 
+ ---
+ 
++_Contributor Covenant Code of Conduct v2.1, from [commit 2038463](https://github.com/EthicalSource/contributor_covenant/blob/8a3be1350b07f38b53bbc7073f765a48c4c53ce1/content/version/2/1/code_of_conduct.md)_
```

Comment:
```suggestion
+_Contributor Covenant Code of Conduct v2.0, from [commit 8001bf8](https://github.com/EthicalSource/contributor_covenant/blob/8001bf8c6e7cd2606657e2816710770d8a79b7dc/content/version/2/0/code_of_conduct.md)_
```

My only remaining change request here is to change this back to pointing at the 2.0 commit from which our current text was originally copied. As discussed previously, our current CoC has not been modified from the upstream; it's the upstream that has since been modified. This diff is currently just undoing those changes.

When we do finally get to applying changes to our CoC, then this needs to be modified, and if we don't have an upstream commit to point at, the patch tool will need a readme or other documentation describing the source and the modifications we apply to it.

Reply:
There will be more changes with v2.0, but I'm force-pushing to make this change so that we can merge this pull request.